third_party/xla/xla/stream_executor/cuda/cuda_blas_lt.cc

```diff
@@ -26,34 +26,21 @@ limitations under the License.
 #if GOOGLE_CUDA
 #include "third_party/gpus/cuda/include/cublasLt.h"
 #include "third_party/gpus/cuda/include/cublas_v2.h"
-<<<<<<< HEAD:tensorflow/compiler/xla/stream_executor/cuda/cuda_blas_lt.cc
 #endif
-#include "tensorflow/compiler/xla/status_macros.h"
-#include "tensorflow/compiler/xla/stream_executor/blas.h"
-#if GOOGLE_CUDA
-#include "tensorflow/compiler/xla/stream_executor/cuda/cuda_blas.h"
-#include "tensorflow/compiler/xla/stream_executor/cuda/cuda_blas_utils.h"
-#else
-#include "tensorflow/compiler/xla/stream_executor/rocm/rocm_blas.h"
-#endif
-#include "tensorflow/compiler/xla/stream_executor/gpu/gpu_activation.h"
-#include "tensorflow/compiler/xla/stream_executor/gpu/gpu_helpers.h"
-#include "tensorflow/compiler/xla/stream_executor/gpu/gpu_stream.h"
-#include "tensorflow/compiler/xla/stream_executor/gpu/gpu_timer.h"
-#include "tensorflow/compiler/xla/stream_executor/scratch_allocator.h"
-#include "tensorflow/compiler/xla/stream_executor/stream.h"
-=======
 #include "xla/status_macros.h"
 #include "xla/stream_executor/blas.h"
+#if GOOGLE_CUDA
 #include "xla/stream_executor/cuda/cuda_blas.h"
 #include "xla/stream_executor/cuda/cuda_blas_utils.h"
+#else
+#include "xla/stream_executor/rocm/rocm_blas.h"
+#endif
```

Comment:
we should remove these rocm code from cuda_blas_lt.cc as https://github.com/openxla/xla/pull/3953 as it doesn't need anymore.